src/main/java/seedu/address/model/person/Time.java

```diff
@@ -0,0 +1,94 @@
+package seedu.address.model.person;
+
+import seedu.address.model.person.exceptions.InvalidTimeException;
+
+/**
+ * Represents the start time, end time, and the date of a task
+ */
+public class Time implements Comparable<Time> {
+    private int hour;
+    private int min;
+
+    /**
+     * Constructor of time.
+     * @param hour The hour part of the time represented by int.
+     * @param min The min part of the time represented by int.
+     * @throws InvalidTimeException Throws when the time is invalid.
+     */
+    public Time(int hour, int min) throws InvalidTimeException {
+        if (hour < 0 || hour >= 24 || min < 0 || min >= 60) {
+            throw new InvalidTimeException();
+        }
+        this.hour = hour;
+        this.min = min;
+    }
```

Comment:
is it possible to use the LocalTime class instead so that we can utilise their functions such as compareTo()?

Reply:
Okay! I'll change it.